cJSON.h

```diff
@@ -81,84 +81,113 @@ typedef struct cJSON_Hooks
       void (*free_fn)(void *ptr);
 } cJSON_Hooks;
 
+#if !defined(__WINDOWS__) && (defined(WIN32) || defined(WIN64) || defined(_MSC_VER) || defined(_WIN32))
+#define __WINDOWS__
+#endif
+#ifdef __WINDOWS__
+#if defined(CJSON_DECLARE_STATIC)
+#define CJSON_DECLARE(type)			type __stdcall
```

Comment:
I know it used to be required.  I'll test if it defaults sanely now.  I think it defaults to cdecl, which may be fine in this case.  I'll review to confirm.  It would be much nicer to be able to do this without needing the arg in the macro.  I know in the past this didn't work.  Are you ok if it works on modern compilers but not older ones?